website/content/api-docs/secret/pki/issuance.mdx

```diff
@@ -804,3 +809,176 @@ $ curl \
+
+This endpoint fetches the current SCEP configuration.
+
+| Method | Path               |
+|:-------|:-------------------|
+| `GET`  | `/pki/config/scep` |
+
+#### Sample request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/pki/config/scep
+```
+
+#### Sample response
+
+```json
+{
+  "data": {
+    "allowed_digest_algorithms": [
+      "sha-256",
+      "sha-384",
+      "sha-512"
+    ],
+    "allowed_encryption_algorithms": [
+      "aes128-cbc",
+      "aes128-gcm",
+      "aes256-cbc",
+      "aes256-gcm"
+    ],
+    "authenticators": {
+      "cert": {
+        "accessor": "auth_cert_598b1030",
+        "cert_role": "scep-ca"
+      },
+      "scep": {
+        "accessor": "auth_scep_797d8401",
+        "scep_role": ""
+      }
+    },
+    "default_path_policy": "sign-verbatim",
+    "enabled": true,
+    "external_validation": {},
+    "last_updated": "2025-05-26T09:59:24-04:00",
+    "return_full_ca_chain": true
+  }
+}
+```
+
+### Set SCEP Configuration <EnterpriseAlert inline="true" />
+
+This endpoint will update SCEP related configuration, returning the
+updated values as a response along with an updated `last_updated` field.
+For a comprehensive overview of SCEP and how to configure it,
+refer to the [SCEP documentation](/vault/docs/secrets/pki/scep).
+
+| Method | Path               |
+|:-------|:-------------------|
+| `POST` | `/pki/config/scep` |
+
+#### Parameters
+
+- `enabled` `(bool: false)` - Specifies whether SCEP is enabled or not.
+
+- `default_path_policy` `(string: "")` - Specifies the behavior for non-role-qualified SCEP requests; can be
+ `sign-verbatim` or a role given by `role:<role_name>`.
+
+- `authenticators` `(map[string]map[string]string: "")` - Specifies the mount accessors SCEP should delegate authentication
+ requests. Map keys can be either `scep` or `cert`, with associated maps containing the key `accessor` with a value
+ containing the auth mount's accessor. For the `scep` type, an optional key `scep_role` parameter is supported which
+ will be passed as the [name](/vault/api-docs/auth/scep#name) parameter during SCEP authentication attempts.
+ For the `cert` type, an optional key `cert_role` parameter is supported which will be passed as the
+ [name](/vault/api-docs/auth/cert#name-6) parameter during certificate authentication attempts.
+
+- `allowed_encryption_algorithms` `([]string)` - the list of allowed encryption algorithms for SCEP requests, allowed
+ values are: `des-cbc`, `3des-cbc`, `aes128-cbc`, `aes256-cbc`, `aes128-gcm`, `aes256-gcm`
+
+- `allowed_digest_algorithms` `([]string)` - the list of allowed digest algorithms for SCEP requests.  Allowed values
+  are: `sha-1`, `sha-256`, `sha-384`, `sha-512`
+
+- `return_full_ca_chain` `(bool: true)` - whether to return the full CA chain within GetCACert responses, this
+  should be set to `false` for compatibility with Microsoft InTune.
```

Comment:
It seems like Microsoft prefers `Intune` to `InTune`. Please change this and the other places where mixed case is used.